.mise.toml

```diff
@@ -0,0 +1,2 @@
+[tools]
+node = "lts"
```

Comment:
Is it possible to make this change as a separate issue or PR?

Reply:
Sure. I've removed it and probably wouldn't bother with a PR to add it.